nathan-shulkin.html

```diff
@@ -8,8 +8,8 @@
   <body>
     <h1>Brief History of CSS</h1>
     <ul>
-      <li><span id="green" class="chrono">1994</span>: The first work on style sheets started at CERN.</div></li>
-      <li><div class="chrono">December 1996</div>: CSS was proposed as W3C standard:
+      <li><span id="green" class="chrono">1994: The first work on style sheets started at CERN.</span></li>
```

Comment:
Your `id` should be placed on the `<li>` element, not your `span`.  Also, the `span` should _not_ incorporate the entire list item - just the actual date (same as your subsequent line items).